src/sentry/SentrySpans.tsx

```diff
@@ -0,0 +1,62 @@
+interface SentrySpanInfo {
+  name: string
+  op: string
+}
+
+export enum SentrySpan {
+  dappkit_connection = 'DappKitConnection',
+  dappkit_transaction = 'DappKitTransaction',
+  fetch_balances = 'FetchBalances',
+  import_contacts = 'ImportContacts',
+  pincode_enter = 'PincodeEnter',
+  pincode_set = 'PincodeSet',
+  send_payment_or_invite = 'SendPaymentOrInvite',
+  send_payment_or_invite_legacy = 'SendPaymentOrInviteLegacy',
+  wallet_connect_connection = 'WalletConnectConnection',
+  wallet_connect_transaction = 'WalletConnectTransaction',
+}
+
+type values = typeof SentrySpan[keyof typeof SentrySpan]
+
+export const SentrySpans: Record<values, SentrySpanInfo> = {
```

Comment:
it took me a while to understand the difference between SentrySpan and SentrySpans, maybe this can be called `SentryTransactionContext` since within Sentry the type of this object is called `TransactionContext`? I'm also a little confused about transaction vs span, these things are really transactions? so we can also rename `SentrySpan` to `SentryTransaction`?

Reply:
I went with the name SentrySpan to match what is seen in the Sentry UI.
![Screen Shot 2022-07-27 at 8 00 06 AM](https://user-images.githubusercontent.com/26950305/181281043-2770da6d-0892-41f4-8d4a-bdd411200665.png)